types/react/index.d.ts

```diff
@@ -36,7 +36,7 @@ type NativeUIEvent = UIEvent;
 type NativeWheelEvent = WheelEvent;
 
 // tslint:disable-next-line:export-just-namespace
-export = React;
+export default React;
```

Comment:
Do not change the declaration file. use `--esModuleInerop` in your project instead. see https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default for more details.